releasenotes/notes/ci-origin-tagging-638df7163e9c4d03.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The pytest plugin now adds a `_dd.origin` tag to every test span for the CI app.
```

Comment:
I actually don't think this needs a release note, this is all internal implementation details (including adding that tag).